src/components/font_size_editor/font_size_editor.ts

```diff
@@ -20,9 +20,12 @@ css/* scss */ `
   .o-font-size-editor {
     height: calc(100% - 4px);
     input.o-font-size {
-      outline-color: ${SELECTION_BORDER_COLOR};
       height: 20px;
       width: 23px;
+
+      &:focus {
+        outline: ${SELECTION_BORDER_COLOR} auto 1px;
+      }
```

Comment:
TBH I think the no outline variant is prettier. I find the blue border quite ugly. But it's less clear that the input is focused. You should ask Francois which we should use.